### DISCUSSION TITLE
How to update an already downloaded subreddit?

### DISCUSSION THREAD

**Discussion by CrunchyJohnson** [MalloyDelacroix/DownloaderForReddit#335] [Q&A] [answered]
If I try to redownload a subreddit I've downloaded before, It just runs for a few seconds and stops. Anyway to download newer posts of a subreddit that's already been downloaded before?

Thanks!

**Comment by MalloyDelacroix** [comment, marked as answer]
The default settings should allow you to continue downloading where a previous download left off.  Check the settings for the subreddit you are trying to download (or the settings for the subreddit list if there is more than one) and make sure that the following settings match:

Score limit eval: NO LIMIT
Limit by date: Checked
Last post date: Checked
Sort posts by: NEW

Also, check the date displayed to the right of the "Last Post Date" radio button and ensure it is the date of the last post you downloaded for that subreddit.  If it is not, you can select Custom Date Limit, and set the date limit to an appropriate date.
